app/models/preneeds/currently_buried_input.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+require 'common/models/base'
+
+module Preneeds
+  class CurrentlyBuriedInput < Common::Base
```

Comment:
Should we just call these as they are in XML? CurrentlyBurried vs CurrentlyBuriedInput? Everything is an input. :)